src/util/featuredQueue.js

```diff
@@ -5,14 +5,24 @@ import { canBeFeatured } from './featuredQueueUtil';
 
 async function rotateFeatured(queue) {
   if (queue.length < 4) {
-    throw new Error(`Not enough cubes in queue to rotate (need 4, have ${queue.length})`);
+    return {
```

Comment:
The code calling rotateFeatured expects a response with messages + succcess to indicate pass/fail, not errors